packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -583,7 +583,7 @@ For example:
 
 ```ts
 new codebuild.Project(stack, 'MyProject', {
-  buildSpec: codebuild.BuildSpec.fromObject({
+  buildSpec: codebuild.BuildSpec.fromObjectToYaml({
```

Comment:
Instead of changing these, can you just add a paragraph in the ReadMe saying something like "If you'd prefer your buildspec to be rendered as YAML in the template, use the `fromObjectToYaml()` method instead, and a simple example?

Reply:
done :+1: